src/chat/summarizers/environments.ts

```diff
@@ -0,0 +1,28 @@
+import { MarkdownString } from "vscode";
+import { Environment } from "../../models/environment";
+
+export function summarizeEnvironment(env: Environment): string {
+  const summary = new MarkdownString()
+    .appendMarkdown(`### "${env.name}"`)
+    .appendMarkdown(`\n- ID: ${env.id}`);
+
+  // only include names and IDs of child resources, for easier follow-up tool calls
+  if (env.kafkaClusters.length) {
+    summary.appendMarkdown(`\n- Kafka Clusters:`);
+    env.kafkaClusters.forEach((cluster) => {
+      summary.appendMarkdown(`\n  - "${cluster.name}" (ID: ${cluster.id})`);
```

Comment:
Knowing the cloud provider + region of Kafka clusters and Flink compute pools might end up being Really Handy.

Reply:
Good call - added in https://github.com/confluentinc/vscode/pull/1755/commits/e993d9663bf457208a7ba26669c2cd56fa115ff2